docs/eventing/broker/kafka-broker.md

```diff
@@ -293,6 +293,35 @@ To change the logging level to `DEBUG`, you need to:
     kubectl rollout restart deployment -n knative-eventing kafka-broker-dispatcher
     ```
 
+## Events Delivery order
+
+When dispatching events, Kafka Broker supports two different ordering guarantees:
+
+* `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
+* `ordered`: Ordered consumer is a per-partition blocking consumer that delivers messages in order.
```

Comment:
we call these "ordering guarantees" above, but then "consumers" here. Can we choose a single term here for referring to this or explain the relationship better?

Reply:
I've added _consumer_ above to refer to the fact that we're talking about the kafka broker consumer delivery guarantees